src/main/java/Duke.java

```diff
@@ -1,11 +1,15 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
 import java.sql.SQLOutput;
 import java.util.Scanner;
 import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.util.regex.Pattern;
 
 public class Duke {
+    static String STR = "------------------------------------------------------------";
     static ArrayList<Task> arr = new ArrayList<Task>();
```

Comment:
Perhaps a more fitting name instead of `arr` will be `tasks` or `taskLists` with emphasis on the plurality since it has a container for tasks.👍